services/models/service.go

```diff
@@ -14,51 +16,68 @@ type ServiceList struct {
 	Services  []ServiceOverview `json:"services"`
 }
 
-type Service struct {
+type ServiceDetails struct {
```

Comment:
What do you think about this change?
The rationale behind it is that now, the WorkloadDetails has a list of services for a given workload. Therefore, if I use the previous Service type, I would list VirtualServices, DestinationRules, Dependencies, Health with empty values.
In consequence, I decided to wrap the actual values of a service in the Service type and have it within this ServiceDetails struct.

Reply:
+1, you could even have one derived from the other and thus not have to duplicate some fields such as Name / Namespace:

```
type ServiceDetails struct {
  	Service
 	Endpoints        Endpoints           `json:"endpoints"`
   	// etc.
}
```